src/spikeinterface/postprocessing/template_similarity.py

```diff
@@ -147,10 +154,144 @@ def _get_data(self):
 compute_template_similarity = ComputeTemplateSimilarity.function_factory()
 
 
+if HAVE_NUMBA:
```

Comment:
When we were making the import faster I remember that we discussed whether this compiles at init time or not. Does anyone have a better memory than me @zm711 @JoeZiminski ?

Reply:
I think the compiling at init is mainly due to type hints. I was actually listening to someone talk about numba the other day and they recommend not to do the type hints for numba and let it infer them. So I think we should be fine. We should see the slow down in the import action right? or we could use tuna (I forget the name @JoeZiminski )

Reply:
Ah yes, it was because Aurelien specified the actual types on the numba decorator. Now I remember.

Reply:
Exactly.